src/coreclr/vm/riscv64/cgencpu.h

```diff
@@ -273,7 +273,7 @@ inline BOOL isJump(PCODE pCode)
 
     TADDR pInstr = PCODEToPINSTR(pCode);
 
-    return *dac_cast<PTR_DWORD>(pInstr) == 0x58000050;
+    return *dac_cast<PTR_DWORD>(pInstr+12) == 0x00000013;
```

Comment:
What is the meaning of `+12` and `0x00000013`?

Reply:
It checks that codes in `pCode` were generated in `emitJump`.
```
inline void emitJump(LPBYTE pBufferRX, LPBYTE pBufferRW, LPVOID target)
{
    LIMITED_METHOD_CONTRACT;
    UINT32* pCode = (UINT32*)pBufferRW;
    // We require 8-byte alignment so the LD instruction is aligned properly
    _ASSERTE(((UINT_PTR)pCode & 7) == 0);
    // auipc ra, 0
    // ld    ra, ra, 16
    // jalr  x0, ra, 0
    // nop    //padding.
    pCode[0] = 0x00000097; // auipc ra, 0
    pCode[1] = 0x0100b083; // ld    ra, 16(ra)
    pCode[2] = 0x00008067; // jalr  x0, ra, 0
    pCode[3] = 0x00000013; // padding nop. Also used for isJump.
    // Ensure that the updated instructions get updated in the I-Cache
    ClrFlushInstructionCache(pBufferRX, 16);
    *((LPVOID *)(pCode + 4)) = target;   // 64-bit target address
}
```

Reply:
It may be nice to write it as `3 * sizeof(UINT32)` to make this a bit more self-describing.

Reply:
Updated. Thank you.